pkg/plugins/policies/meshaccesslog/plugin/v1alpha1/plugin.go

```diff
@@ -232,16 +234,13 @@ func applyToGateway(
 }
 
 func configureInbound(
-	fromRules core_rules.Rules,
+	conf *api.Conf,
```

Comment:
I'd remove the pointer here (and `if conf == nil {` check as well). When `configureInbound` it's called from `applyToGateway` check the `conf == nil` there

Reply:
changed